cmd/vgmanager/vgmanager.go

```diff
@@ -216,7 +219,18 @@ func run(cmd *cobra.Command, _ []string, opts *Options) error {
 		return fmt.Errorf("unable to set up ready check: %w", err)
 	}
 
-	if err := mgr.AddHealthzCheck("healthz", healthz.Ping); err != nil {
+	if err := mgr.AddHealthzCheck("healthz", func(req *http.Request) error {
+		if len(lvmdConfig.DeviceClasses) == 0 {
+			log.FromContext(req.Context()).Error(ErrNoDeviceClassesAvailable, "not healthy",
+				"available_device_classes", lvmdConfig.DeviceClasses)
```

Comment:
We already know that len(lvmdConfig.DeviceClasses) = 0, so why do we want to print it here?